ee/libcglue/src/glue.c

```diff
@@ -453,8 +484,18 @@ int chdir(const char *path) {
 }
 #endif
 
-#ifdef F_mkdir
-int mkdir(const char *path, mode_t mode) {
+#ifdef F_fchdir
+int fchdir(int fd)
+{
+	errno = ENOSYS;
+	return -1;
+}
+#endif
+
+#ifdef F__mkdir
+// Called from newlib mkdirr.c
```

Comment:
I think then we must put `-DHAVE_MKDIR` in the options of compiling `newlib`

Reply:
Yeah that might be better

Reply:
How do you want to do this? are you going a create to newlib too? I would like to merge this PR.

Reply:
I made a newlib https://github.com/ps2dev/newlib/pull/16